crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -641,6 +641,7 @@ where
                                 package_id: o.id(),
                                 move_package: p.clone(),
                                 checkpoint_sequence_number,
+                                previous_transaction: o.previous_transaction,
```

Comment:
Note to self -- this change is no longer necessary (we no longer need to track the previous transaction), so I need to remove it.